src/Components/test/E2ETest/Tests/EventTest.cs

```diff
@@ -117,6 +116,40 @@ public void MouseDownAndMouseUp_CanTrigger()
             Browser.Equal("onmousedown,onmouseup,", () => output.Text);
         }
 
+        [Fact]
+        public void PointerDown_CanTrigger()
+        {
+            MountTestComponent<MouseEventComponent>();
+
+            var input = Browser.FindElement(By.Id("pointerdown_input"));
+
+            var output = Browser.FindElement(By.Id("output"));
+            Assert.Equal(string.Empty, output.Text);
+
+            var actions = new Actions(Browser).ClickAndHold(input);
+
+            actions.Perform();
+            Browser.Equal("onpointerdown", () => output.Text);
+        }
+
+        [Fact]
+        public void DragDrop_CanTrigger()
+        {
+            MountTestComponent<MouseEventComponent>();
+
+            var input = Browser.FindElement(By.Id("drag_input"));
+            var target = Browser.FindElement(By.Id("drop"));
+
+            var output = Browser.FindElement(By.Id("output"));
+            Assert.Equal(string.Empty, output.Text);
+
+            var actions = new Actions(Browser).DragAndDrop(input, target);
+
+            actions.Perform();
+            // drop doesn't seem to trigger in Selenium. But it's sufficient to determine "any" drag event works
```

Comment:
FYI, I can see this working when I run it in a browser (Chrome, Edge, FireFox,), but not using Selenium. I don't see anything obviously wrong with the codez,, but like the comment says if we can verify any drag event works (in this case it's the dragstart) it's sufficient.